internal/search/job/jobutil/job.go

```
@@ -195,19 +195,20 @@ func NewBasicJob(inputs *search.Inputs, b query.Basic) (job.Job, error) {
 		}
 	}
 
-	{
-		// Apply search result sanitize rules post-filter
-		var sanitizePatterns []*regexp.Regexp
-		c := conf.Get()
-		if c.ExperimentalFeatures != nil && c.ExperimentalFeatures.SearchSanitizePatterns != nil {
-			for _, val := range c.ExperimentalFeatures.SearchSanitizePatterns {
-				if re, err := regexp.Compile(val); err == nil {
-					sanitizePatterns = append(sanitizePatterns, re)
+	{ // Apply search result sanitization post-filter if enabled
+		if inputs.SanitizeSearch {
+			var sanitizePatterns []*regexp.Regexp
+			c := conf.Get()
+			if c.ExperimentalFeatures != nil && c.ExperimentalFeatures.SearchSanitization != nil {
+				for _, val := range c.ExperimentalFeatures.SearchSanitization.SanitizePatterns {
+					if re, err := regexp.Compile(val); err == nil {
+						sanitizePatterns = append(sanitizePatterns, re)
```

We should probably surface this error somewhere. Right now, if an admin puts an invalid pattern, it will just silently do nothing.